public/addCareer.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Path-18</title>
+  </head>
+  <body>
+
+    <h2 class="form-title">Please add a career here</h2>
+
+    <form class="form" action="/add-career" method="post">
+      <div class="form-arabic">
+        <label for="title-ar">Career title (Arabic):</label>
+        <input type="text" name="title_ar" id="title-ar">
+        <label for="tagline-ar">Career tagline (Arabic):</label>
+        <input type="text" name="tagline_ar" id="tagline-ar">
+        <label for="description-ar">Description (Arabic):</label>
+        <input type="text" name="description_ar" id="description-ar">
+      </div>
+
+      <div class="form-hebrew">
+        <label for="title-he">Career title (Hebrew):</label>
+        <input type="text" name="title_he" id="title-he">
+        <label for="tagline-ar">Career tagline (Hebrew):</label>
+        <input type="text" name="tagline_he" id="tagline-he">
+        <label for="description-ar">Description (Hebrew):</label>
+        <input type="text" name="description_he" id="description-he">
+      </div>
+
+      <div class="general-inputs">
+        <label for="salary-start">Junior salary:</lable>
+        <input type="text" name="salary_start" id="salary-start">
+        <label for="salary-ten-year">Senior salary:</label>
+        <input type="text" name="salary_ten_year" id="salary-ten-year">
+        <label for="image-url">Image:</label>
+        <input type="text" name="image_url" id="image-url">
+        <label for="icon-url">Icon:</label>
+        <input type="text" name="icon_url" id="icon-url">
+        <br>
+      </div>
+
+      <div class="universities">
+        <h3 class="form-title">Universities</h3>
+        <input type="checkbox" id="haifa">
+        <label for="haifa">Haifa University</label>
+        <label for="bagrut-haifa">Required Bagrut/Tawjihi Grade:</label>
+        <input type="text" id="bagrut-haifa" name="grade_bagrut">
+        <label for="psyc-haifa">Required Psychometric Grade:</label>
+        <input type="text" id="psyc-haifa" name="grade_psychometric">
+        <br>
```

Comment:
I'm not a fan of `<br>`s, I'd say wrapping each section in a `section` or even using a [fieldset](https://www.w3schools.com/tags/tag_fieldset.asp)